apps/src/blockly/addons/cdoGesture.js

```diff
@@ -55,28 +33,47 @@ export function overrideHandleTouchMove(blocklyWrapper) {
   };
 
   /**
-   * Custom method: Create a block dragger and start dragging the selected block.
+   * Update this gesture to record whether a block is being dragged from the
+   * flyout.
+   * This function should be called on a pointermove event the first time
+   * the drag radius is exceeded.  It should be called no more than once per
+   * gesture. If a block should be dragged from the flyout this function creates
+   * the new block on the main workspace and updates targetBlock_ and
+   * startWorkspace_.
+   *
+   * @returns True if a block is being dragged from the flyout.
+   * @override Initiates duplication of shadow argument reporter blocks.
    */
-  cdoGesture.startDraggingBlock = function () {
+  cdoGesture.updateIsDragging = function (e) {
     // Begin Customization
     // Create a new block and set it as the target block for the drag.
     if (this.shouldDuplicateOnDrag) {
       this.duplicateOnDrag();
     }
     // End Customization
 
-    const BlockDraggerClass = Blockly.registry.getClassFromOptions(
-      Blockly.registry.Type.BLOCK_DRAGGER,
-      this.creatorWorkspace.options,
-      true
-    );
+    if (!this.startWorkspace_) {
+      throw new Error(
```

Comment:
what happens if this error is thrown? do we get a crash or just an error in the console?

Reply:
I'm not sure, but I would expect a crash. This is actually Blockly's code copied directly, so I wasn't think we needed to test it too much.

https://github.com/google/blockly/blob/803dcfba25a830960513a53bcb2dbfe9b69a656f/core/gesture.ts#L334